### ISSUE TITLE
Add QR code to PDFs for verification

### ISSUE THREAD

**Issue by alexfigtree** [digitalcredentials/learner-credential-wallet#604]
**Is your feature request related to a problem? Please describe.**
When a PDF is generated from the wallet there is no way to identify the PDF as being sourced from a Verifiable Credential. 

**Describe the solution you'd like**
The LCW has the ability to generate a link to verifier plus for the credential. We could use this functionality to add a QR code to the PDF that will contain that link.  We could consider asking the user if they would like a url added to their PDF.

**Describe alternatives you've considered**
We could also consider using CBOR (?) to pack the data into the PDF and signing it but that's probably outside the scope of this because a link to verifier plus is far more usable. 

**Additional context**
One of the challenges is that we provide the user with an option to create and destroy a URL. Do we use this url in the PDF and have them create it before exporting or do we keep this functionality separate? Also, links stored in the verifier plus db expire in one year. If we add a url to the PDF, would it be. different one every time the PDF is generated

**Comment by kayaelle** [comment]
Added a priority label because issuing of diplomas is imminent and we will want the PDFs to be verifiable.

**Comment by alexfigtree** [comment]
Team meeting: https://docs.google.com/document/d/1cxKjKE6AfHxvPxYVYkgs6sXrs2Wk5OywWNAGwy1NXls/edit

**Comment by alexfigtree** [comment]
On hold until team meets again

**Comment by alexfigtree** [comment]
Currently blocked by: https://github.com/digitalcredentials/web-verifier-plus/issues/126

**Comment by alexfigtree** [comment]
Dev team meeting: Have it function in the same way as the Post to LinkedIn option

**Comment by alexfigtree** [comment]
Implementation details for this ticket: For wallet on share screen:
1) Check if credential is verified, then
2) Check if credential has renderMethod field, then
3) Check if this method has html embedded or a link to an html file
4) If there is custom html provided by the issuer via a template, in the export to PDF code, look through html for a qr code tag provided by the DCC (let's call it `qr_code`), so that if we search for `{qr_code}`, we replace what's in the curly braces with an img element of a QR code of the share link.

Goal is to replace any QR codes provided in a custom html template, NOT to add them.

**Comment by alexfigtree** [comment]
Updated title to reflect work done on PDF fix; QR code implementation now implemented in #650.
